asm/assemble.py

```diff
@@ -430,13 +430,17 @@ def free_space_org_list_sorter(code_chunk_tuple):
                     "-o",
                     bin_name,
                 ]
+
+                if result := call(["cargo", "fmt"], cwd="./custom-functions"):
+                    raise Exception("Formatting rust functions failed.")
+
                 # add custom functions from rust
                 if is_custom_function and file_path == "main.dol":
```

Comment:
```suggestion
                # add custom functions from rust
                if is_custom_function and file_path == "main.dol":
                    if result := call(["cargo", "fmt"], cwd="./custom-functions"):
                        raise Exception("Formatting rust functions failed.")
```
Otherwise it formats for *every* asm patch, which is unnecessary

Reply:
Ah yeah, good catch